node/coinstacks/bitcoin/pulumi/Pulumi.sample.yaml

```diff
@@ -20,10 +20,16 @@ config:
     ## api - if specified an api service will be deployed
     api:
       enableDatadogLogs: false
+      cpuLimit: '1'
```

Comment:
Sample values are meant to be "dev minimums" to have low overhead running out of the box after copy/paste